hcaptcha/api/hcaptcha.api

```diff
@@ -0,0 +1,103 @@
+public final class com/stripe/hcaptcha/BuildConfig {
+	public static final field BUILD_TYPE Ljava/lang/String;
+	public static final field DEBUG Z
+	public static final field LIBRARY_PACKAGE_NAME Ljava/lang/String;
+	public fun <init> ()V
+}
+
+public final class com/stripe/hcaptcha/HCaptcha$Companion {
+	public final fun getClient (Landroidx/fragment/app/FragmentActivity;)Lcom/stripe/hcaptcha/HCaptcha;
+	public final fun getClient (Landroidx/fragment/app/FragmentActivity;Lcom/stripe/hcaptcha/config/HCaptchaInternalConfig;)Lcom/stripe/hcaptcha/HCaptcha;
+}
+
+public final class com/stripe/hcaptcha/HCaptchaDialogFragment$Companion {
+	public final fun newInstance (Lcom/stripe/hcaptcha/config/HCaptchaConfig;Lcom/stripe/hcaptcha/config/HCaptchaInternalConfig;Lcom/stripe/hcaptcha/HCaptchaStateListener;)Lcom/stripe/hcaptcha/HCaptchaDialogFragment;
+	public final fun test ()Lcom/stripe/hcaptcha/HCaptchaDialogFragment;
+}
+
+public final class com/stripe/hcaptcha/HCaptchaError$Companion {
+	public final fun fromId (I)Lcom/stripe/hcaptcha/HCaptchaError;
+}
+
+public final class com/stripe/hcaptcha/HCaptchaHtmlKt {
```

Comment:
You should be able to remove this by adding a `@file:RestrictTo` to the top of the file.

Reply:
I've tried that, but it doesn't seem to affect the api dump.

Reply:
I pushed a commit that fixes this.